### PR TITLE
Account for stats with no newline

### PR DESCRIPTION
**Pull request by RDruon**
See new fixture/snapshot

### CODE REVIEW

**Review by jgrund — CHANGES_REQUESTED** [review]
rustfmt

File: src/fixtures/valid/valid.txt [hunk 1]
```diff
@@ -252,14 +252,7 @@ mds_connect               44 samples [usec] 10 1774 4146 3299378
 mds_statfs                64446 samples [usec] 6 87 1723266 50620388
 obd_ping                  4 samples [usec] 10 16 50 644
 out_update                23 samples [usec] 7 118 879 59435
-ost.OSS.ost_seq.stats=
```
Comment (jgrund) [inline src/fixtures/valid/valid.txt:255]: Instead of removing valid data, add a new entry that shows new behavior.

Comment (RDruon) [reply src/fixtures/valid/valid.txt]: I added a new fixture and kept the existing one intact

File: src/fixtures/valid/valid_no_newline.txt [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+snapshot_time             1689060419.303818818 secs.nsecs
+req_waittime              239924 samples [usec] 6 80519 23284390 432682600300
+req_qdepth                239924 samples [reqs] 0 1 189 189
+req_active                239924 samples [reqs] 1 3 282653 368119
+req_timeout               239924 samples [sec] 1 15 3597962 53968262
+reqbuf_avail              500525 samples [bufs] 63 64 32023877 2048915579
+ost_statfs                239924 samples [usec] 3 95307 11187378 188117229936
+ost.OSS.ost_io.stats=snapshot_time             1695133177.877704894 secs.nsecs
+ost.OSS.ost_out.stats=snapshot_time             1695133177.877719213 secs.nsecs
+ost.OSS.ost_seq.stats=snapshot_time             1695133177.877732780 secs.nsecs
```
Comment (RDruon) [inline src/fixtures/valid/valid_no_newline.txt:236]: No  newline here

**Pull request merged**